src/Artsy/Router/Utils/__tests__/findCurrentRoute.test.ts

```diff
@@ -0,0 +1,54 @@
+import { findCurrentRoute } from "../findCurrentRoute"
+import { Match, Router, Location } from "found"
+
+describe("findCurrentRoute", () => {
+  const getMatch = (config: Partial<Match>): Match => ({
+    routes: [],
+    router: ({} as unknown) as Router,
+    context: {},
+    routeIndices: [0],
+    location: ({} as unknown) as Location,
+    params: {},
+    ...config,
+  })
+
+  it("should return a base nested route", () => {
+    const match = getMatch({
+      routeIndices: [0, 0],
+      routes: [
+        {
+          path: "/artist/:artistID",
+          children: [
+            {
+              path: "/",
+            },
+          ],
+        },
+      ],
+    })
+    expect(findCurrentRoute(match)).toHaveProperty("path", "/")
+  })
+
+  it("should return a non-zero nested route", () => {
+    const match = getMatch({
+      routeIndices: [0, 2],
+      routes: [
+        {
+          path: "/artist/:artistID",
```

Comment:
Can we add one more test that asserts against 

```
/artist/:artistID/baz
```
I'm not aware of any routes that are more than three paths deep but this would cover all of our bases.

Reply:
Wait, `/artist/:artistID/cv` is tested here. Do you want one a level deeper?

Reply:
baz (with children)

Reply:
Okay, I added a test for `/artist/:artistID/foo/bar` and `/bar` has children that we _don't_ want to match against.